src/lxc/console.c

```diff
@@ -260,16 +260,21 @@ int lxc_setup_tios(int fd, struct termios *oldtios)
 
 	newtios = *oldtios;
 
-	/* Remove the echo characters and signal reception, the echo
-	 * will be done with master proxying */
-	newtios.c_iflag &= ~(IGNBRK|ISTRIP|INLCR|IGNCR|ICRNL|IXON|IXANY|IXOFF);
-	newtios.c_iflag &= BRKINT;
```

Comment:
Thanks - not sure what I prefer;  I'd like the cfmakeraw for less cryptic code (and less chance of accidental breakage later as we're setting fewer of our ownoptions), but ssh is a nice standard to go by.

I'll merge as is - thanks.